src/db/sql/sqlUpdateBuilder.ts

```diff
@@ -0,0 +1,52 @@
+import { Objects } from '@openforis/arena-core'
+import { SqlBuilder } from './sqlBuilder'
+import { Table } from '../table'
+import { Column } from '../column'
+
+type setType = { column: Column | string; value: string }
```

Comment:
preferably types, interfaces, classes uppercase. better to call this SetValue ?

Reply:
Updated case, `Type` postfix refers that it's the functions type